galera/src/replicator_smm.cpp

```diff
@@ -260,7 +260,7 @@ galera::ReplicatorSMM::ReplicatorSMM(const struct wsrep_init_args* args)
 
     log_debug << "End state: " << uuid << ':' << seqno << " #################";
 
-    update_state_uuid (uuid);
+    update_state_uuid (uuid, seqno);
```

Comment:
Please add relevant comment. It would be bit confusing for end-user to see why the same value is being set again in the grastate.dat.